Tests iOS/MyList/MyListTests.swift

```diff
@@ -164,6 +164,8 @@ class MyListTests: XCTestCase {
                 return Response.myList()
             } else if apiRequest.isForTags {
                 return Response.emptyTags()
+            } else if apiRequest.isForSlateLineup {
```

Comment:
I don't think you need to add the isForSlateLineup. You're not testing that request in this test.